FeedStoreChallenge/Cache.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  Cache.swift
+//  FeedStoreChallenge
+//
+//  Created by Livia Vasconcelos on 19/06/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(Cache)
+internal class Cache: NSManagedObject {
```

Comment:
The convention in this codebase is to omit the default access level keyword of `internal`, please remove it from this class and its properties and methods.
Also, `Cache` is not supposed to be subclassed, so it should be marked as `final`.
```suggestion
final class Cache: NSManagedObject {
```

Reply:
Fixed!

Reply:
Same as the `CachedImage`, please remove the `internal` keyword from properties and methods of `Cache`.